src/machines/java/com/enderio/machines/EIOMachines.java

```diff
@@ -32,6 +34,8 @@ public static void onConstruct(FMLConstructModEvent event) {
         MachineLang.register();
         MachineRecipes.register();
         MachineCapacitorKeys.register();
+
+        MinecraftForge.EVENT_BUS.register(FluidTankBlockEntity.class);
```

Comment:
Can you use the annotations instead of registering manually here please

Reply:
Fixed in 7de7cbaaa8a9c24b9bea48518feedc2b6fafac79 and 11535800ec28e4b58dcd4a4123647bda116bc039